unpub/lib/src/package_store.dart

```diff
@@ -5,7 +5,7 @@ abstract class PackageStore {
     throw 'downloadUri not implemented';
   }
 
-  Stream<List<int>> download(String name, String version) {
+  Future<Stream<List<int>>> download(String name, String version) async {
```

Comment:
It's a breaking change. Can this `Future` be converted into a `Stream`?

Reply:
I was having trouble doing this because I'm just newer to handling dart futures.  In s3_store.dart I used waitFor().  Its marked as experimental in Dart docs, so if there is a better way to handle it please let me know.